components/brave_vpn/resources/panel/components/general.ts

```diff
@@ -39,17 +39,25 @@ export const Title = styled.h1`
 
 export const ButtonText = styled.button`
```

Comment:
Optional: You might be able to replace this with the `plain` or `plain-faint` button from Nala